README.md

```diff
@@ -1036,6 +1037,10 @@ can pass additional arguments that control the code output:
   function expressions. See
   [#640](https://github.com/mishoo/UglifyJS2/issues/640) for more details.
 
+- `wrap_func_args` (default `false`) -- pass `true` to wrap function expressions
+  that are passed as arguments. See
+  [OptimizeJS]https://github.com/nolanlawson/optimize-js) for more details.
```

Comment:
You missed parenthesis😉 
`[OptimizeJS](https://github.com/nolanlawson/optimize-js)`